src/components/PaginationFooter/PaginationFooter.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import FooterWrapper from './FooterWrapper';
+import PaginationContent from './PaginationContent';
+import type { FooterWrapperProps, PaginationContentProps } from './types';
+
+interface Props extends Omit<FooterWrapperProps, 'children'>, PaginationContentProps {}
+
+const PaginationFooter = (props: Props) => (
```

Comment:
Now only exposes a single component. So no need to wrap two components to render pagination.